daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
@stevvooe Thanks for the review. Regarding the `pkg/` structure, we wanted to follow the structure that `docker\docker` adopted : https://github.com/docker/docker/blob/master/pkg/README.md. The idea is that all of the packages under the `pkg` are utility packages used by libnetwork without specific to its internals and follows the same reasoning :
> Utility packages are kept separate from the docker core codebase to keep it as small and concise as possible. If some utilities grow larger and their APIs stabilize, they may be moved to their own repository under the Docker organization, to facilitate re-use by other projects. However that is not the priority.

Hence, we would like to keep that convention. But yes, we will fix the indexing issue that you brought up and that will make it much more clear.